docs/production-deployment/cloud/migrate/automated.mdx

```diff
@@ -192,7 +192,8 @@ There are two options for switching Temporal clients to the cloud:
      automatically forward from your server to the cloud.
   4. Complete migration. Your self-hosted Temporal clients will no longer receive any tasks from your server, allowing you to stop these clients.
 
-- Option 2 (not recommended)- Deploy one set of Temporal clients and switch the namespace endpoint during migration. With this option, your workflows will stop making progress.
+- Option 2 - Deploy one set of Temporal clients and switch the namespace endpoint during migration. With this option, if your workers are misconfigured during the switch, then it is
```

Comment:
i would rephrase something like, "make sure to test your workers configuration before the switch to make sure there is no disruption of the your workflow execution"